openwisp_notifications/websockets/handlers.py

```diff
@@ -11,12 +16,20 @@ def notification_update_handler(reload_widget=False, notification=None, recipien
         notification = NotificationListSerializer(notification).data
     except (NotFound, AssertionError):
         pass
+    is_notification_storm = reload_widget and (
+        Notification.objects.filter(
+            recipient=recipient,
+            timestamp__gte=now() - timedelta(10),
+        ).count()
+        > 10
+    )
```

Comment:
A notification storm is observed when more than 10 notifications are created for a user in last 10 seconds. (1 notification/second). 

I used this value from my personal experience of using OpenWISP. I have seen checks failing for a couple of devices at a time. So to avoid catching such occurrences as "notification storm", I used above value.